tokenizers/src/tokenizer/added_vocabulary.rs

```diff
@@ -192,6 +192,14 @@ impl AddedVocabulary {
         &self.added_tokens_map
     }
 
+    /// Get only the added tokens
+    pub fn get_added_tokens(&self) -> Vec<String> {
+        self.added_tokens
+            .iter()
+            .map(|token| token.content.clone())
+            .collect()
+    }
```

Comment:
Let's remove this. This is creating an entire new vec, which isn't necessary.

Reply:
Removed it, totally agree